### PR TITLE
Installer

### PR DESCRIPTION
**Pull request by brmcdoug**
moved install script stuff into jalapeno-installer directory.  some fixes so builds work with new project structure. also a lsnode bugfix

### CODE REVIEW

**Review by sbezverk — CHANGES_REQUESTED** [review]
Please change .gitignore to ignore `.DS_Store` it should not appear in a public repo

File: .gitignore [hunk 1]
```diff
@@ -1,2 +1,23 @@
...
+*.test
+
+# Output of the go coverage tool, specifically when used with LiteIDE
+*.out
+
+# Dependency directories (remove the comment below to include it)
+# vendor/
+
+# .DS_Store
+.DS_Store
```
Comment (justaugustus) [inline .gitignore:23]:
```suggestion
.DS_Store

```

nit: add newline to EOF

You'll also want to make sure that anything matching .gitignore that was already committed is removed e.g., https://github.com/cisco-open/jalapeno/blob/0a94ef83c5c95705d5a566c82a480accdd995c97/.DS_Store

File: linkstate-edge/arangodb/arangodb.go [hunk 2]
```diff
@@ -116,8 +116,8 @@ func (a *arangoDB) StoreMessage(msgType dbclient.CollectionType, msg []byte) err
 	switch msgType {
 	case bmp.LSLinkMsg:
 		return a.lsLinkHandler(event)
-	case bmp.LSNodeMsg:
-		return a.lsNodeHandler(event)
+		// case bmp.LSNodeMsg:
+		// 	return a.lsNodeHandler(event)
```
Comment (justaugustus) [inline linkstate-edge/arangodb/arangodb.go:120]:
```suggestion
```

Remove extraneous commented-out code.

**Review by justaugustus — CHANGES_REQUESTED** [review]
A few nits...

Also, consider renaming `jalapeno-installer/` to something simpler that doesn't repeat.

The current path would be: https://github.com/cisco-open/jalapeno/jalapeno-installer
What about `deploy/` or `install/`?

File: linkstate-edge/arangodb/arangodb.go (hunk 2 above)
Comment (brmcdoug) [reply linkstate-edge/arangodb/arangodb.go]: thanks!  i also made a couple tweaks per request from sbezverk

**Pull request merged**